packages/web-components/src/components/card/__stories__/card.stories.ts

```diff
@@ -0,0 +1,76 @@
+/**
+ * @license
+ *
+ * Copyright IBM Corp. 2020
+ *
+ * This source code is licensed under the Apache-2.0 license found in the
+ * LICENSE file in the root directory of this source tree.
+ */
+
+import { boolean } from '@storybook/addon-knobs';
+import { html } from 'lit-element';
+import ArrowRight20 from 'carbon-web-components/es/icons/arrow--right/20';
+import '../card-footer';
+import '../../image/image';
+import ifNonNull from 'carbon-web-components/es/globals/directives/if-non-null';
+import readme from './README.stories.mdx';
+import styles from './card.stories.scss';
+import textNullable from '../../../../.storybook/knob-text-nullable';
+import '../card';
+
+export const Default = ({ parameters }) => {
+  const { image, href, alt, defaultSrc, eyebrow, heading, copy, inverse, footer } = parameters?.props?.Card ?? {};
+  return html`
+    <dds-card color-scheme=${inverse ? 'inverse' : ''} href=${ifNonNull(href || undefined)}>
+      ${image
+        ? html`
+            <dds-image slot="image" alt="${ifNonNull(alt)}" default-src="${ifNonNull(defaultSrc)}"></dds-image>
+          `
+        : html``}
+      <slot slot="eyebrow">${eyebrow}</slot>
+      <slot slot="heading">${heading}</slot>
+      ${copy
+        ? html`
+            <p>${copy}</p>
+          `
+        : html``}
```

Comment:
Given there is no variable part in the template here:

```suggestion
        : ''}
```

An alternative is using `nothing` from `lit-html`: https://lit-html.polymer-project.org/guide/writing-templates#rendering-nothing